src/content/en/2021/compression.md

```diff
@@ -184,20 +191,24 @@ Gzip compression is applied largely around compression level 6, extending to lev
 {{ figure_markup(
   image="compression-levels-gzip.png",
   caption="Compression levels for Gzip.",
-  description="This breaks down the usage of compression levels of responses using Gzip, for compression level 1 to 9, as well as `opt` which corresponds to optimizers such as Zopfli. It shows peaks at level 1 and at level 6, as well as a smaller peak at level 9. Each level is also broken down by content type.",
+  description="Stacked bar charts showing the usage of compression levels of responses using Gzip, for compression level 1 to 9, as well as `opt` which corresponds to optimizers such as Zopfli. It shows peaks at level 1 and at level 6, as well as a smaller peak at level 9. Each level is also broken down by content type.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQtfyTM9VEweN_Hli3IuxxqU1CRap4V5Q28baEs7aEBResoPRgk9Dwp1m_vdS9lzNlfO8J4hZN7GPT7/pubchart?oid=586666706&format=interactive",
   sheets_gid="150560131"
   )
 }}
 
-## How to analyze compression on your sites
+{# TODO - should data be broken down more to say which content type? #}
```

Comment:
The figure has the breakdown, so I take this to mean to describe it in text. I've done so, with a focus on JavaScript since the main difference is visible there.

However in doing so I found an error in the analysis, there was a tiny json file that skewed the Gzip 'optimal' stats. I've fixed the data, figure, and updated the text for this, as well as the reference png image. For Brotli this made no difference.